packages/xo-web/src/xo-app/jobs/schedules/index.js

```diff
@@ -209,12 +213,23 @@ export default class Schedules extends Component {
   ]
 
   render() {
-    const { cronPattern, jobs, schedule, schedules, timezone } = this.state
+    const {
+      cronPattern,
+      enabled,
+      jobs,
+      schedule,
+      schedules,
+      timezone,
+    } = this.state
     const userData = { jobs }
     return (
       <div>
         <h2>{_('newSchedule')}</h2>
         <form id='newScheduleForm'>
+          <div className='form-group'>
+            <label>{_('scheduleEnableAfterCreation')}</label>{' '}
+            <Toggle onChange={this.toggleState('enabled')} value={enabled} />
```

Comment:
What about [this](https://github.com/vatesfr/xen-orchestra/pull/5111#discussion_r446074962)?